assignments1/unflatten.js

```diff
@@ -0,0 +1,27 @@
+function merge(ans,arr,val)
+{
+	var len=arr.length-1;
+	for(var iter=0;iter<len;iter++)
+	{
```

Comment:
start the block with { in the same line only like:
`for(var iter = 0; iter < len; iter++) {`